### PR TITLE
add seminar-notes directory

### PR DESCRIPTION
**Pull request by bennn**
Yesterday, Jan suggested we put notes for the PRL seminar "somewhere online". This is a proposal for how that would work.

@maxsnew is it okay with you if we put these notes on the PRL website?

- Save notes to `seminar-notes/<ANCHOR>.md`, where `<ANCHOR>` is the anchor for the notes used on the `seminars.rkt` page (see [L63](https://github.com/nuprl/website/blob/334f956b108cd92dc31628ec0791f30ec42f0fb5/seminars.rkt#L63) for the anchor to Max's seminar)
- Keep notes in plain-text Markdown. (Rendering the notes seems like too much work for everyone involved.)
- The seminars page will display a link to the notes -- if they exist -- after you click the seminar's title.

For example, here's my local "seminars.html" page:
screen shot 2016-12-16 at 16 13 45

And after I click Max's title:
screen shot 2016-12-16 at 16 13 25

Very low-key.

### CODE REVIEW

**Comment by bennn** [comment]
Any comments on the "license" [file](https://github.com/nuprl/website/blob/334f956b108cd92dc31628ec0791f30ec42f0fb5/seminar-notes/README.md) would be very useful.

**Comment by gasche** [comment]
Comment, I am fine with the content, but instead of "Please email" I would say "Feel free to email".

**Comment by maxsnew** [comment]
I am fine with putting the notes online, but there are some errors in the notes that need to be fixed first.

File: seminar-notes/new-retractionsblame.md [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+Max:
+  right, for the same reason that non-empty intersection isn't transitive
+
+Here's the typing rule for an explicit cast from type `A` to type `B`:
+
+```
+    Γ ⊢ t : A
+    A ~ B
+    ---
+    Γ ⊢ <A ⇒ B>p t : B
```
Comment (maxsnew) [inline seminar-notes/new-retractionsblame.md]: The ⇒ should go in the opposite direction to match their syntax.

File: seminar-notes/new-retractionsblame.md [hunk 2]
```diff
@@ -0,0 +1,657 @@
...
+  Maybe this one is insufficent to justify blame.
+M:
+  exactly, you need the 2 for the blame part
+Max:
+  Well you still have 2 functions, so still possible to distinguish
+  But there you really need 2 contracts
+M:
+  maybe we can build more
+Max:
+  maybe you have more junk
```
Comment (maxsnew) [inline seminar-notes/new-retractionsblame.md]: zing!

File: seminar-notes/new-retractionsblame.md [hunk 3]
```diff
@@ -0,0 +1,657 @@
...
+    ⊤ -> B ⇒ Dyn ⇒ N -> ⊥
+```
+
+(BG: maybe,
+
+```
+   ((λ (x : Bool)
+     (printN (<Dyn ⇒ Nat>(<Bool ⇒ Dyn> x))))
+    true)
+```
```
Comment (maxsnew) [inline seminar-notes/new-retractionsblame.md]: why the lambda instead of beta reducing?

Comment (bennn) [inline seminar-notes/new-retractionsblame.md]: To make a client/server distinction. The `x : Bool` specification is important to me.

**Comment by bennn** [comment]
better now?

**Comment by maxsnew** [comment]
Thanks, Ben, great work here!
Just that one markdown issue and otherwise it's good to go.

**Comment by bennn** [comment]
Merged!
https://github.com/nuprl/website/commit/6c7215867352051b8336bc2b0b74471701acb0c6

**Pull request closed without merging**